projects/ion/src/lib/input-select/input-select.component.html

```diff
@@ -41,6 +43,22 @@
           [(ngModel)]="value"
           (ngModelChange)="handleChange()"
         />
+        <ion-icon
```

Comment:
You could create an ng-template to these icons, so you can reuse them here and below.